tensorflow-core-ops/tensorflow-core-ops.cabal

```diff
@@ -16,7 +16,7 @@ library
   exposed-modules:  TensorFlow.GenOps.Core
   autogen-modules:  TensorFlow.GenOps.Core
   build-depends:  bytestring
-                , proto-lens >= 0.4.0 && < 0.6.0
```

Comment:
Could you retain the safety net for future incompatible changes by changing to keep < 0.7.0 everywhere?

Reply:
Good point: Done. (By fixing to 0.6.*.)